config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -44,6 +44,41 @@ presubmits:
           requests:
             cpu: 7
             memory: 12Gi
+  - name: pull-kubernetes-verify-strict-lint
+    cluster: k8s-infra-prow-build
+    always_run: true
+    decorate: true
+    # This entire job is currently experimental. It it turns out to be useful,
+    # the job can be removed and the same check can run as part of pull-kubernetes-verify
+    # by removing the exclusion of verify-golangci-lint-pr.sh in hack/make-rules/verify.sh.
+    optional: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
+    annotations:
+      fork-per-release: "true"
+      testgrid-create-test-group: 'true'
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+        imagePullPolicy: Always
+        command:
+        - runner.sh
+        args:
+        - /bin/sh
+        - "if [ -x hack/verify-golangci-lint-pr.sh ]; then hack/verify-golangci-lint-pr.sh; fi"
+        resources:
+          # Consider reducing memory limits after govet memory usage issue is
+          # addressed in https://github.com/kubernetes/kubernetes/issues/93822
```

Comment:
This was copy-and-pasted from "pull-kubernetes-verify". It's not directly applicable, but I am not sure which limits to pick instead.

Even for "pull-kubernetes-verify" the limits may soon be different (["go vet" run differently](https://github.com/kubernetes/kubernetes/pull/116166) , or are already (the [referenced issue](https://github.com/kubernetes/kubernetes/issues/93822) is fixed).